volmaster/daemon.go

```diff
@@ -70,25 +70,33 @@ func (d *DaemonConfig) Daemon(listen string) {
 	r := mux.NewRouter()
 
 	postRouter := map[string]func(http.ResponseWriter, *http.Request){
-		"/request":      d.handleRequest,
-		"/create":       d.handleCreate,
-		"/copy":         d.handleCopy,
-		"/mount":        d.handleMount,
-		"/mount-report": d.handleMountReport,
-		"/unmount":      d.handleUnmount,
-		"/remove":       d.handleRemove,
+		"/config/global":                    d.handleGlobalUpload,
```

Comment:
We don't need a namespace for this as there's only one. Let's avoid adding anything to this route's text.